packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1505,10 +1506,10 @@ export interface ControlBarProps {
 }
 
 // @public
-export const createAzureCommunicationCallAdapter: ({ userId, displayName, credential, locator, alternateCallerId }: AzureCommunicationCallAdapterArgs) => Promise<CallAdapter>;
+export const createAzureCommunicationCallAdapter: ({ userId, displayName, credential, locator, alternateCallerId, roleHint }: AzureCommunicationCallAdapterArgs) => Promise<CallAdapter>;
 
 // @public
-export const createAzureCommunicationCallAdapterFromClient: (callClient: StatefulCallClient, callAgent: CallAgent, locator: CallAdapterLocator) => Promise<CallAdapter>;
+export const createAzureCommunicationCallAdapterFromClient: (callClient: StatefulCallClient, callAgent: CallAgent, locator: CallAdapterLocator, roleHint?: Role | undefined) => Promise<CallAdapter>;
```

Comment:
The arguments to this factory function are getting out of hands...

Really it should have been:

`createAzureCommunicationCallAdapterFromClient(callClient, callAgent, options)` with the clients explicitly listed and then all options in an option bag.

But we already added the `locator` as a top-level arg here.

I'd recommend still introducing an options bag now and then taking it to ARB to see what they think: 
`createAzureCommunicationCallAdapterFromClient(callClient, callAgent, locator, options)`